html/elements/img.json

```diff
@@ -264,7 +264,7 @@
                 "version_added": true
               },
               "safari": {
-                "version_added": false
+                "version_added": "11.1"
               },
               "safari_ios": {
                 "version_added": false
```

Comment:
The post you linked to also reported that this feature is available for Safari for iOS as well:

```suggestion
                "version_added": "11.3"
```

Reply:
Hi @ddbeck ! Thank you for your follow up.

Reply:
Thank you for the PR. Do you plan to apply the change?

Reply:
I'm sorry. I didn't notice the button.